packages/lib/versionInfo.ts

```diff
@@ -32,6 +52,14 @@ export default function versionInfo(packageInfo: any) {
 		console.info(gitInfo);
 	}
 
+	const plugins = getPluginsArr(PluginService.instance().plugins);
```

Comment:
From where versionInfo() is called, you already have the list of plugins in `props.plugins`, so you don't need to load the PluginService here. Just pass props.plugins to this function.